### PR TITLE
McAfee Web Gateway: add smart-descriptions

### PR DESCRIPTION
**Pull request by squioc**
Add smart-descriptions for McAfee Web Gateway

### CODE REVIEW

File: events/smart-descriptions.json [hunk 1]
```diff
@@ -3655,5 +3655,90 @@
         "value": "changed-audit-configuration"
       }]
     }
+  ],
+  "mcafee web gateway": [
+    {
+      "value": "Connection from {source.ip} to {url.domain} for user {user.name} blocked: {event.reason}",
```
Comment (PierrickV) [inline events/smart-descriptions.json:3661]:
```suggestion
      "value": "Connection from {source.ip} to {url.domain} initiated by user {user.name} blocked: {event.reason}.",
```

File: events/smart-descriptions.json [hunk 2]
```diff
@@ -3655,5 +3655,90 @@
...
+      }
+      ],
+      "relationships": [{
+        "source": "source.ip",
+        "target": "url.domain",
+        "type": "denied"
+      }]
+    },
+    {
+      "value": "Connection from {source.ip} to {url.domain} for user {user.name} allowed",
```
Comment (PierrickV) [inline events/smart-descriptions.json:3689]:
```suggestion
      "value": "Connection from {source.ip} to {url.domain} initiated by user {user.name} allowed.",
```

File: events/smart-descriptions.json [hunk 3]
```diff
@@ -3655,5 +3655,90 @@
...
+      }
+      ],
+      "relationships": [{
+        "source": "source.ip",
+        "target": "url.domain",
+        "type": "denied"
+      }]
+    },
+    {
+      "value": "Connection from {source.ip} to {url.domain} blocked: {event.reason}",
```
Comment (PierrickV) [inline events/smart-descriptions.json:3676]:
```suggestion
      "value": "Connection from {source.ip} to {url.domain} blocked: {event.reason}.",
```

File: events/smart-descriptions.json [hunk 4]
```diff
@@ -3655,5 +3655,90 @@
...
+      }
+      ],
+      "relationships": [{
+        "source": "source.ip",
+        "target": "url.domain",
+        "type": "allowed"
+      }]
+    },
+    {
+      "value": "Connection from {source.ip} to {url.domain} allowed",
```
Comment (PierrickV) [inline events/smart-descriptions.json:3704]:
```suggestion
      "value": "Connection from {source.ip} to {url.domain} allowed.",
```

File: events/smart-descriptions.json [hunk 5]
```diff
@@ -3655,5 +3655,90 @@
...
+      {"field": "event.reason"}
+      ],
+      "relationships": [{
+        "source": "source.ip",
+        "target": "url.domain",
+        "type": "error"
+      }]
+    },
+    {
+      "value": "The verification of the connection from {source.ip} to {url.domain} failed for an undefined reason",
```
Comment (PierrickV) [inline events/smart-descriptions.json:3731]:
```suggestion
      "value": "The verification of the connection from {source.ip} to {url.domain} failed for an undefined reason.",
```

File: events/smart-descriptions.json [hunk 6]
```diff
@@ -3655,5 +3655,90 @@
...
+      }
+      ],
+      "relationships": [{
+        "source": "source.ip",
+        "target": "url.domain",
+        "type": "allowed"
+      }]
+    },
+    {
+      "value": "The verification of the connection from {source.ip} to {url.domain} failed: {event.reason}",
```
Comment (PierrickV) [inline events/smart-descriptions.json:3717]:
```suggestion
      "value": "The verification of the connection from {source.ip} to {url.domain} failed: {event.reason}.",
```

File: events/smart-descriptions.json (hunk 2 above)
Comment (squioc) [reply events/smart-descriptions.json]: Partly applied

File: events/smart-descriptions.json (hunk 3 above)
Comment (squioc) [reply events/smart-descriptions.json:3676]: I don't think the dot at the end of the string is required.

File: events/smart-descriptions.json (hunk 4 above)
Comment (squioc) [reply events/smart-descriptions.json:3704]: I don't think the dot at the end of the string is required.

File: events/smart-descriptions.json (hunk 5 above)
Comment (squioc) [reply events/smart-descriptions.json:3731]: I don't think the dot at the end of the string is required.

File: events/smart-descriptions.json (hunk 6 above)
Comment (squioc) [reply events/smart-descriptions.json:3717]: I don't think the dot at the end of the string is required.

**Review by PierrickV — APPROVED** [review]

**Pull request merged**